src/Components/BeastImage.js

```diff
@@ -0,0 +1,64 @@
+//import Image from 'react-bootstrap/Image';
+import { Component } from 'react';
+import SelectedBeast from './SelectedBeast';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import BeastData from '../data.json'
+
+
+
+class BeastImage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      status: "Vote", 
+      show: false,  
+      selectedBeast: {},
+      data: BeastData   
+    }
+  }
+
+  handleClick = (event) => {
+    // toggle the status between Yay and Nay
+    this.setState({
+      status: this.state.status === "Like" ? "Dislike" : "Like"
+    });
+    this.openModal(this.props.bio.title)
```

Comment:
Feedback here: I think I commented on this in previous lab. Instead of `status` in state, I would have `count` or `clicks` etc.

Then you setState call would increment the clicks. Grab a TA to get ya goin' on this.

 For lab 2, a heart should be displayed beside the click count. Every time a user clicks on the image (lab 2, clicks/count is incremented) and shown beside this heart.

For lab 3, when user clicks on an image, the modal should open and display that particular beast image :) You will need to do some additional refactoring here as you need to send a function into your `Main` component that allows the user to update state in the App.